stores/amazon_requests.py

```diff
@@ -180,22 +180,26 @@ def __init__(
 
         # Initialize proxies for stock check session:
         # Assuming same username/password for all proxies
-        # username = [INSERT USERNAME HERE]
-        # password = [INSERT PASSWORD HERE]
-        #
 
-        # self.proxies = [
-        #     {
-        #       'http': f"http://{username}:{password}@X.X.X.X:XXXX",
-        #       'https': f"http://{username}:{password}@X.X.X.X:XXXX",
-        #     },
-        #     {
-        #       'http': f"http://{username}:{password}@X.X.X.X:XXXX",
-        #       'https': f"http://{username}:{password}@X.X.X.X:XXXX",
-        #     },
-        # ]
+        # load dictionary from the json
+        proxy_dict = json.load(open("config/amazon_requests_proxies.json"))
+
+        username = proxy_dict["user"]
+        password = proxy_dict["pass"]
+        enable_proxy = proxy_dict["enable"]
 
+        # build the proxies list
         self.proxies = []
+        for i in proxy_dict["ip_port"]:
```

Comment:
This is not performant. Check for enabled True and move statements in if clause

Reply:
I have updated the code to use flag now since I was able to figure that out in cli.py, I'm sorry I'm not sure exactly what you mean. The for loop builds the proxy dictionaries so not really sure what you mean by non performant.

Sorry I have only been working with Python for 2 months and has been a few years since I have coded before that